lib/mesos/util.py

```diff
@@ -78,6 +78,18 @@ def format_commands_help(cmds):
 
     return help_string
 
+def format_flags_help(flags):
```

Comment:
Done!
I also moved the help flag addition within the function to give a more consistent formatting with the rest of the code.